worker/buildbot_worker/test/unit/test_scripts_base.py

```diff
@@ -33,7 +34,10 @@ class TestIsWorkerDir(misc.FileIOMixin, misc.StdoutAssertionsMixin,
 
     def setUp(self):
         # capture output to stdout
-        self.mocked_stdout = io.BytesIO()
+        if PY3:
```

Comment:
I prefer when you say something that you say exactly what you want.
I can't guess what's in your mind, and don't like submitting multiple guesses of what you want and getting it wrong.